dut/drivers.wake

```diff
@@ -0,0 +1,107 @@
+
+tuple BasicDriverPlan =
+  global DUHDocument:         Path
+  global OMFile:              Unit => Path
+  global VendorName:          String
+  global DeviceName:          String
+  global LocalMetalDirectory: String
+  global DUTName:             String
```

Comment:
I'm a bit confused by this `DUTName` argument. Why is this necessary, and why are we doing string matching on it? Why can't we automatically use the same driver for any DUT as long as the driver, not the DUT, matches?

I think I'm starting to realize I don't really understand at a high level how this mechanism is intended to work. Can you explain the relationships between a hardware IP block (also known as a "device"), a driver, a DUT, and a BasicDriverPlan? In particular, it seems like right now a BasicDriverPlan is one-to-one with the cross product of IP blocks and DUTs, which doesn't sound like it'll scale. How do you create multiple drivers for one DUT? How do you create a driver for one block that can be reused in multiple DUTs?